019-ThisOldDataWarehouse/Coach/Solutions/Challenge2/Readme.md

```diff
@@ -6,10 +6,17 @@
+The objective of this challenge is to build a Data Lake with Azure Data Lake Store (ADLS) Gen 2.  The Data Lake will be a staging area where all our source system data files reside. We need to ensure this Data Lake is well organized and doesn't turn into a swamp. This challenge will help us organize the folder structure and setup security to prevent unauthorized access.  Lastly, we will extract data from the WWI OLTP platform and store it in the Data Lake.  
+
+The OLTP platform is on-premise so you will need to build a hybrid archtiecture to integrate it into Azure.  Keep in mind that the pipeline that you build will become the <b>EXTRACT</b> portion of the new E-L-T process.  Based on requirements for this new process, you will need to be sure that changes can be captured on a daily basis.  Stored procedures have already been compiled in the source OLTP database, but they will require data parameters in order to be executed.  This means that the new pipeline will need to be able to generate and pass those parameters to the source.  The first requirement is to build a functional POC that is able to move a single dataset to the new ADLS Gen 2 data lake. Ideally, it would be nice to make the process table driven so that new pipelines do not need to be created for each additional table that needs to be copied. (Optional, sharing to give insights on end-state).
+
+There are many adventures you can take on this Challenge and one of them can be to setup Synapse Analytics Workspace.  This will setup attached ADLS Gen2 storage along with the data pipelines.  You don't have to wait till Challenge 3 to get started.  When Synapse is GA, this would be the ideal path.
 
 
 ## Environment Setup
+
+<b>Note: </b> Until Synapse Analytics goes GA, the coach's notes and students guides will leverage the terms Azure Data Lake Store Gen2, Azure Data Factory and Azure Synapse Database.  These terms will be replaced with Linked Storage, Data Pipelines and SQL Pools respectively as the reference documentation is updated upon GA.  It is acceptable to use Synapse Analytics Workspace as one of the adventures. We did not explicitly mention it since supporting documentation is missing.
```

Comment:
Instead of using the b tags, you can put two asterisks wrapping "Note:" on the left and right.

Reply:
Added astericks for bold